imports/api/resource/server/publications.js

```diff
@@ -100,6 +100,7 @@ Meteor.publish('resourcesSearch', function(orgId, searchStr='', limit=50, fromTi
             created: 1,
             updated: 1,
             searchableData: 1,
+            data: 1
```

Comment:
Unfortunately it can be too resource intensive to send the `data` to the client on pages like this. Some of the yamls can get pretty big. And thats part of the reason we made the `searchableData`. So for this case, we'd want to have razeedash-api put the commit into in searchableData and then use that for displaying on the pages